databases/week2/hyf-task.sql

```diff
@@ -0,0 +1,75 @@
+-- Part1
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+    task(title, description, created, updated, due_date, status_id, user_id)
+VALUES
+    ('Bake muffins', 'For 5 people', '2020-10-04 03:16:42', '2020-10-05 16:56:58', '2020-11-05 16:50:50', 2, 6);
+    
+-- Change the title of a task
+UPDATE
+    task
+SET title = 'Iron clothes'
+WHERE id = 2;
+-- Change a task due date
+UPDATE 
+    task
+SET due_date = '2021-10-04 03:16:42'
+WHERE id = 3;
+-- Change a task status
+UPDATE 
+    task
+SET status_id = 1
+WHERE id = 4;
+-- Mark a task as complete
+UPDATE 
+    task
+SET status_id = 3
+WHERE id = 5;
+-- Delete a task
+DELETE 
+FROM
+    task
```

Comment:
Unnecessary line break for both FROM and WHERE.

Pointer: improves readability and clarity

Reply:
see other occurrences and update